public/views/Search.vue

```diff
@@ -393,6 +409,15 @@ export default Vue.extend({
         : result.dataset.variables.length ?? 0;
     },
 
+    getStorageNameFromResult(result: any) {
```

Comment:
`any` shouldn't be used here.  I know it was in the existing code, but given that this file is being modified anyways, we should clean it up.  I was curious to see how much work it would entail, so I did a quick clean up on another branch that I can merge into this one if you want. See 7b523c027e54e03525244754db0200f9ea6a9490.  I dealt with a few linting issues as well.

Reply:
Yeah, I'll merge that in.

Reply:
Merged your changes back to this branch after double checking them on my local, I think it's ready to merge back main now.